### PR TITLE
Issue #9 - Open Source Guidelines for Gimlet

### PR DESCRIPTION
**Pull request by sourabhsingh282**
This PR is referenced with issue #9 and as part of the issue, this PR adds some nice contributing guidelines to the first-time contributors. 
 
Reference: https://github.com/gimlet-io/gimlet-cli#contribution-guidelines

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,3 +31,26 @@ helm install my-release onechart/onechart \
 ```
 
 See all [Examples](/examples/deploying-an-image)
+
+## Contribution Guidelines
+
+Thankyou for your interest in contributing to the Gimlet project.
```
Comment (laszlocph) [inline README.md:37]: Small typo: "Thank you"

File: README.md [hunk 2]
```diff
@@ -31,3 +31,26 @@ helm install my-release onechart/onechart \
...
+
+Below are some of the guidelines and best practices for contributing to this repository:
+
+### Issues
+
+If you encounter any issues or have any relevant questions, please add an issue to [GitHub issues](https://github.com/gimlet-io/onechart/issues/new). Make sure to include the detailed description report with descriptive title.
+
+Also, if you have already implemented the solution or want to help us with the pull request, go ahead with pull request.
```
Comment (laszlocph) [inline README.md:45]: Please change this line to: 

> If you are running a fork of OneChart and would like to upstream a feature, please open a pull request for it.
>
> If you are planning to add a new feature to OneChart, please open an issue for it first. Helm charts are prone to having too many features, and OneChart want to keep the supported use-cases in-check. Proposed features have to be generally applicable, targeting newcomers to the Kubernetes ecosystem.

**Review by laszlocph — CHANGES_REQUESTED** [review]
Thank you very much for your contribution! This is your first in OneChart, kudos :tada:

There are a couple of recommendations for your PR, I'm happy to merge it, once we cleared those.

Hope to see you around!

**Comment by sourabhsingh282** [comment]
Thanks @laszlocph for the review, i have updated the pull request :)

**Comment by soniasingla** [comment]
@sourabhsingh282 this is amazing, thanks for your contribution, hope to see you around! 🤗

**Pull request merged**